src/views/open-api-docs-view/components/open-api-operations/index.tsx

```diff
@@ -36,9 +40,17 @@ export function OpenApiOperations({
 									<OperationHeader
 										className={s.header}
 										slug={operation.slug}
-										headerText={operation.operationId}
+										headingAriaLabel={operation.operationId}
+										headingSlot={addWordBreaks(
```

Comment:
`addWordBreaks` and `splitOnCapitalLetters` helpers are where the magic kinda happens, though hopefully the function names make it feel like the very straightforward non-magic that it actually is!